src/rpc/methods/sync.rs

```diff
@@ -220,6 +221,7 @@ mod tests {
             mpool: Arc::new(pool),
             bad_blocks: Default::default(),
             sync_state: Arc::new(parking_lot::RwLock::new(Default::default())),
+            event_handler: Arc::new(EthEventHandler::new()),
```

Comment:
Perhaps a more suiting name would be an `eth_event_handler` for the field.

Reply:
corrected